### PR TITLE
.github: add pull request template

### PR DESCRIPTION
**Pull request by Piraty**

### CODE REVIEW

**Comment by fosslinux** [comment]
Suggestion: by default comment out the part that only should be used when travis isn't used? And then let people uncomment it when they need it? I feel like it commented-out should be the default as for the majority of PRs travis will be used.

File: .github/pull_request_template.md [hunk 1]
```diff
@@ -0,0 +1,19 @@
+<!-- Mark items with [x] where applicable -->
+#### Have the results of the proposed changes been tested?
+- [ ] I use the packages affected by the proposed changes on a regular basis and confirm this PR works for me
+- [ ] I generally don't use the affected packages but briefly tested this PR
+- [ ] I did not test this PR
```
Comment (ahesford) [inline .github/pull_request_template.md:5]: Remove this option. Being forced to leave the test options blank without testing might cause some people to do a bit more testing before submitting PRs.

File: .github/pull_request_template.md [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+
+<!-- If Travis cannot be used to validate the build result (for example if the
+build exceeds the timecap of 45min), make sure to put "[ci skip]" in the HEAD
+commit, which skips travis builds completely (saves CI resources). Uncomment the
+following section in this case. -->
```
Comment (ahesford) [inline .github/pull_request_template.md:10]: > If GitHub CI cannot be used to validate the build result (for example, if the build is likely to take several hours), make sure to include "[ci skip]' in either the PR title or this description. When skipping CI, uncomment and complete the following section. Note: for builds that are likely to complete in less than XXX hours, it is not acceptable to skip CI.

(I don't know the time limit, but XXX needs to be filled in.)

File: .github/pull_request_template.md [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+<!--
+#### Does it build successfully? (Only if builds take too long for CI)
```
Comment (ahesford) [inline .github/pull_request_template.md:12]: > Does it build and run successfully? (Please choose at least one native build and, if supported, at least one cross build. More are better.)

**Review by ahesford — CHANGES_REQUESTED** [review]

File: .github/pull_request_template.md (hunk 2 above)
Comment (ericonr) [reply .github/pull_request_template.md:10]: The hard limit is 6 hours.

**Comment by ericonr** [comment]
How do you feel about adding an item for "if this is a new package, explain why it should be packaged"?

File: .github/pull_request_template.md (hunk 2 above)
Comment (Piraty) [reply .github/pull_request_template.md]: thanks

File: .github/pull_request_template.md (hunk 3 above)
Comment (Piraty) [reply .github/pull_request_template.md]: thanks!

**Comment by Piraty** [comment]
> How do you feel about adding an item for "if this is a new package, explain why it should be packaged"?

what do you expect people to fill in? `i use this package, so i want Void to ship it` ?

**Comment by ericonr** [comment]
Maybe a better wording would be "how does the package fit our packaging guidelines?" ?

**Comment by Piraty** [comment]
how about ?
> this new package meets the [quality requirements](https://github.com/void-linux/void-packages/blob/master/Manual.md#quality-requirements)

**Comment by ericonr** [comment]
Sounds good.

**Comment by Piraty** [comment]
next round of review

**Comment by the-maldridge** [comment]
I am relatively satisfied with this PR.  If there are no further concerns I will merge it on Friday.

**Pull request merged**